src/v3/src/components/PasswordRequirements/PasswordRequirements.tsx

```diff
@@ -39,22 +38,23 @@ const PasswordRequirements: UISchemaElementComponent<{
 }> = ({ uischema }) => {
   const { data } = useWidgetContext();
   const {
-    fieldKey,
```

Comment:
sorry, I think I was meaning to remove the targetKey..

also, possible to rename `fieldKey` -> `fieldName`, seems like to be consistent with other types

Reply:
Hmm, not sure I understand here, you want to keep this key? This is just so the requirements component knows which input to run validation on, but I removed it because it is only one of two possible options so no need for the transformer to send that value.